### DISCUSSION TITLE
How do I ignore two different files from the command-line?

### DISCUSSION THREAD

**Discussion by Zamiell** [nadeesha/ts-prune#178] [Q&A]
If I have two files of `foo.ts` and `bar.ts`, and I want to ignore `bar.ts`, then I can perform the following `ts-prune` command:

```
npx ts-prune --ignore bar.ts
```

This works great.

However, if I have three files of `foo.ts` and `bar.ts` and `baz.ts`, and I want to ignore `bar.ts` and `baz.ts`, then how do I do this?

**Comment by Zamiell** [comment]
Oh, looks like [this PR](https://github.com/nadeesha/ts-prune/pull/177/files) answers the question. Thank you @calebpeterson !

**Reply by calebpeterson** [reply]
Happy to help @Zamiell! Hopefully we can get it merged soon so the answer is easier to find for the next person 👍
